include/dogecoin/dogecoin.h

```diff
@@ -133,7 +133,7 @@ typedef uint8_t dogecoin_bool; //!serialize, c/c++ save bool
 #define P2PKHLEN 35
 #define PRIVKEYHEXLEN DOGECOIN_ECKEY_PKEY_LENGTH * 2 + 1
 #define PUBKEYHEXLEN 67
-#define PUBKEYHASHLEN 41
+#define PUBKEYHASHLEN 51 // 40 + 6 + 4 + 1
```

Comment:
i think this was meant for uint160 pubkey hash which if memory serves correct was 20 + 1 terminator? if so maybe we can add another?